### ISSUE TITLE
ymin += 0.2 * (ymax - ymin + 1)

### ISSUE THREAD

**Issue by LiuDHS** [zisianw/FaceBoxes.PyTorch#43]
Hello, can you tell me why you use the following code when calculating ymin in test.py:
`ymin += 0.2 * (ymax - ymin + 1)`
Thank you!

**Comment by zisianw** [comment]
Hi @LiuDHS, it is done following test script [here](https://github.com/sfzhang15/FaceBoxes/tree/master/test) and are used in AFW and PASCAL only. I run an experiment without the 0.2 using pretrained model, in AFW accuracy remains unchanged while PASCAL drops slightly by 0.08%.
The prior boxes in FaceBoxes use aspect ratio:1 only, you could possibly use other or multiple aspect ratio to account various sizes better.
